components/browser/menu2/src/main/java/mozilla/components/browser/menu2/ext/PopupWindow.kt

```diff
@@ -23,24 +23,27 @@ internal fun PopupWindow.displayPopup(
     containerView.measure(spec, spec)
 
     val (availableHeightToTop, availableHeightToBottom) = getMaxAvailableHeightToTopAndBottom(anchor)
+    val (availableWidthToLeft, availableWidthToRight) = getMaxAvailableWidthToLeftAndRight(anchor)
     val containerHeight = containerView.measuredHeight
 
     val fitsUp = availableHeightToTop >= containerHeight
     val fitsDown = availableHeightToBottom >= containerHeight
+    val reversed = availableWidthToLeft < availableWidthToRight
```

Comment:
Could we have issues with right to left locales?

Reply:
From my testing there's no issue.  When we detect right to left, we use the opposite (mirroring) animation.

Reply:
👍🏽

Reply:
I'll add some RTL tests.  Thanks